### PR TITLE
Added missing bower dependancies.

### PR DESCRIPTION
**Pull request by lawkaringithi**

### CODE REVIEW

File: app/index.html [hunk 1]
```diff
@@ -11,7 +11,11 @@
 
     <link rel="stylesheet" href="assets/vendor.css">
     <link rel="stylesheet" href="assets/web-client.css">
-    <link rel="stylesheet" href="styles/app.css">
+    <link rel="stylesheet" href="assets/app.css">
```
Comment (bomb1e) [inline app/index.html]: Is the change from styles/ to assets/ standard/ ?

File: app/styles/how-it-works.css [hunk 2]
```diff
@@ -0,0 +1,5 @@
+/* How it works stying */
```
Comment (bomb1e) [inline app/styles/how-it-works.css]: Ah ok. So this compiles to /assets

Comment (lawkaringithi) [inline app/styles/how-it-works.css]: Yea, it's mapped to assets

File: tests/integration/components/how-it-works-test.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import { moduleForComponent, test } from 'ember-qunit';
```
Comment (bomb1e) [inline tests/integration/components/how-it-works-test.js]: And the tests? ember-cli generator? Pretty cool

**Pull request merged**